app/views/api/v0/classified_listings/index.json.jbuilder

```diff
@@ -0,0 +1,36 @@
+json.array! @classified_listings do |listing|
+  json.type_of              "listings"
+  json.id                   listing.id
+  json.title                listing.title
+  json.slug                 listing.slug
+  json.body_markdown        listing.body_markdown
+  json.bumped_at            listing.bumped_at
+  json.cached_tag_list      listing.cached_tag_list
+  json.category             listing.category
+  json.contact_via_connect  listing.contact_via_connect
+  json.processed_html       listing.processed_html
+  json.published            listing.published
+  json.slug                 listing.slug
+  json.last_buffered        listing.last_buffered
+  json.tag_list             listing.tag_list
+
+  json.user do
+    json.name             listing.user.name
```

Comment:
you can assign the user to a variable, this code is ruby, so you don't keep accessing it.

better yet, there's a partial you can use to render this block: https://github.com/thepracticaldev/dev.to/blob/master/app/views/api/v0/articles/_user.json.jbuilder

with something like:

```ruby
json.partial! PATH, user: listing.user
```